### PR TITLE
aqd:py: ensure that notifyd uses the same Python interpreter as aqd

### LINKED ISSUE #76: notifyd should be started with the same interpreter as aqd
`notifyd` is started by `aqd` but without specifying an explicit interpreter. As a result the default system interpreter is used (the one returned by `env python`) and this breaks the support for virtualenv. Instead `aqd` should start `notify` using its current interpreter (`sys.executable`).

### PR DESCRIPTION
**Pull request by jouvin**
Required for proper support of virtualenv.

Fixes #76

### CODE REVIEW

File: lib/twisted/plugins/aqd.py [hunk 1]
```diff
@@ -232,7 +232,9 @@ def stop_coverage():
                                        "aq_notifyd")
                 if not os.path.exists(notifyd):
                     notifyd = notifyd + ".py"
-                args = [notifyd, "--config", config.baseconfig]
```
Comment (urbonegi) [inline lib/twisted/plugins/aqd.py:235]: We ensure this by modifying the shebang in the aq_notifyd and aqd to have exact version of python interpreter. This is done by Makefile when we build release. But i see your point. As the sys.executable would give the value of a python that aqd.py run on and we do not do different versions for the 2 processes - i see no problem. Btw, one always can set 'run_aqnotifyd' config to false and run/start the aq_notifyd as separate service. @gombasg  do you see any issues with merge this one in?

Comment (jouvin) [reply lib/twisted/plugins/aqd.py:235]: @urbonegi the point, as you understood, is that if you want to use virtualenv, you cannot resolve the path at build time. As the goal is to ensure that you use the same interpreter for boht, I don't see how my modification will create a problem!
setting `run_aqnotifyd`to false is just adding the management burden when it is so convenient to have the broker taking the control of everything.
I hope @gombasg will agree!

**Comment by urbonegi** [comment]
@jouvin we will release the PR this week. All fine :)

**Comment by urbonegi** [comment]
Merged

**Pull request closed without merging**